neural_compressor/torch/utils/environ.py

```diff
@@ -91,6 +91,9 @@ def get_torch_version():
     return version
 
 
+GT_TORCH_VERSION_2_3_2 = get_torch_version() > Version("2.3.2")
```

Comment:
How about following transformers design? `check_min_torch_version("2.3.2")` or `require_version("torch>=2.3.2", "log")`
```
# Will error if the minimal version of Transformers is not installed. Remove at your own risks.
check_min_version("4.42.0.dev0")

require_version("datasets>=2.14.0", "To fix: pip install -r examples/pytorch/language-modeling/requirements.txt")
```

Reply:
Actually, this is a temporary flag check. We can remove it after the Torch 2.3.2 or 2.4 release, which is coming soon.